src/DssSpell.t.sol

```diff
@@ -496,4 +307,43 @@ contract DssSpellTest is DssSpellTestBase {
         }
```

Comment:
`test_auth` and `test_auth_in_sources` should be enabled as there are new chainlog entries.

Reply:
ah good catch - enabled